sources/packages/web/src/components/partial-view/DesignationAgreement/DesignationAgreementSummary.vue

```diff
@@ -36,6 +33,7 @@
           <template #body="slotProps">
             <v-btn
               outlined
+              :color="COLOR_BLUE"
               @click="goToViewDesignation(slotProps.data.designationId)"
```

Comment:
not part of this PR, actually, we can avoid line 89 to 91 and emit event directly by ` @click="$emit('viewDesignation',slotProps.data.designationId)"